tests/TestCase.php

```diff
@@ -209,23 +208,28 @@ protected function getCollectionName(): string
      */
     protected function getInvalidArrayValues(bool $includeNull = false): array
     {
-        return array_merge([123, 3.14, 'foo', true, new stdClass()], $includeNull ? [null] : []);
+        return [123, 3.14, 'foo', true, new stdClass(), ...($includeNull ? [null] : [])];
```

Comment:
Reformatting data generator using array unpacking.

Reply:
I feel bad for forgetting PHP has this feature now 😅